tripy/tripy/constants.py

```diff
@@ -16,6 +16,8 @@
 #
 
 CONSTANT_IR_PRINT_VOLUME_THRESHOLD = 5
+STROGE_OP_PRINT_VOLUME_THRESHOLD = 64
```

Comment:
We should rename this since it's not really about printing. Maybe print->cache?

Reply:
changed